conda/base/context.py

```diff
@@ -963,6 +966,7 @@ def category_map(self):
             "Channel Configuration": (
                 "channels",
                 "channel_alias",
+                "channel_settings",
```

Comment:
Can we add this to `"Hidden and Undocumented"` for now (maybe with a `# Once this is used, move it to "Channel Configuration".` or similar comment)?